pkg/config/get.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+func pathSplit(r rune) bool {
+	return r == '.' || r == '[' || r == ']' || r == '"'
+}
+
+func GetValueFromConfig(stringPath string, object interface{}) (interface{}, error) {
+	keyPath := strings.FieldsFunc(stringPath, pathSplit)
+	// log.Println(stringPath)
```

Comment:
You could remove this commented out debugging printf :)

Reply:
Oups 😮